hw/ip/sysrst_ctrl/doc/dv/index.md

```diff
@@ -0,0 +1,123 @@
+
+### Alert_agents
+SYSRST_CTRL testbench instantiates (already handled in CIP base env) [alert_agents]({{< relref "hw/dv/sv/alert_esc_agent/README.md" >}}):
+[list alert names].
+The alert_agents provide the ability to drive and independently monitor alert handshakes via alert interfaces in SYSRST_CTRL device.
+
+### UVC/agent 1
+[Describe here or add link to its README]
+
+### UVC/agent 2
+[Describe here or add link to its README]
+
+### UVM RAL Model
+The SYSRST_CTRL RAL model is created with the [`ralgen`]({{< relref "hw/dv/tools/ralgen/README.md" >}}) FuseSoC generator script automatically when the simulation is at the build stage.
+
+It can be created manually by invoking [`regtool`]({{< relref "util/reggen/README.md" >}}):
+
+### Reference models
+[Describe reference models in use if applicable, example: SHA256/HMAC]
+
+### Stimulus strategy
+#### Test sequences
+The test sequences reside in `hw/ip/sysrst_ctrl/dv/env/seq_lib`.
+All test sequences are extended from `sysrst_ctrl_base_vseq`, which is extended from `cip_base_vseq` and serves as a starting point.
+It provides commonly used handles, variables, functions and tasks that the test sequences can simple use / call.
+Some of the most commonly used tasks / functions are as follows:
+* task 1:
+* task 2:
+
+#### Functional coverage
+To ensure high quality constrained random stimulus, it is necessary to develop a functional coverage model.
+The following covergroups have been developed to prove that the test intent has been adequately met:
+* cg1:
+* cg2:
+
+### Self-checking strategy
+#### Scoreboard
+The `sysrst_ctrl_scoreboard` is primarily used for end to end checking.
+It creates the following analysis ports to retrieve the data monitored by corresponding interface agents:
+* analysis port1:
+* analysis port2:
+<!-- explain inputs monitored, flow of data and outputs checked -->
+
+#### Assertions
+* TLUL assertions: The `tb/sysrst_ctrl_bind.sv` file binds the `tlul_assert` [assertions]({{< relref "hw/ip/tlul/doc/TlulProtocolChecker.md" >}}) to the IP to ensure TileLink interface protocol compliance.
+* Unknown checks on DUT outputs: The RTL has assertions to ensure all outputs are initialized to known values after coming out of reset.
+* assert prop 1:
+* assert prop 2:
+
+## Building and running tests
+We are using our in-house developed [regression tool]({{< relref "hw/dv/tools/README.md" >}}) for building and running our tests and regressions.
+Please take a look at the link for detailed information on the usage, capabilities, features and known issues.
+Here's how to run a smoke test:
+```console
+$ $REPO_TOP/util/dvsim/dvsim.py $REPO_TOP/hw/ip/sysrst_ctrl/dv/sysrst_ctrl_sim_cfg.hjson -i sysrst_ctrl_smoke
+```
+
+## Testplan
+<!-- TODO: uncomment the line below after adding the testplan.
+Please make sure the testplan is added to `/util/build_docs.py`. -->
```

Comment:
The presubmit checks are failing with this message:
```
ERROR 2021/11/18 14:41:03 HTML for testplan has not been generated for /hw/ip/sysrst_ctrl/data/sysrst_ctrl_testplan.hjson (looked at build/docs-generated/hw/ip/sysrst_ctrl/data/sysrst_ctrl_testplan.hjson.testplan)
Total in 2323 ms
Error: Error building site: "/home/vsts/work/1/s/hw/ip/sysrst_ctrl/doc/dv/index.md:123:1": failed to render shortcode "incGenFromIpDesc": failed to process shortcode: "/home/vsts/work/1/s/site/docs/layouts/shortcodes/incGenFromIpDesc.html:30:3": execute of template failed: template: shortcodes/incGenFromIpDesc.html:30:3: executing "shortcodes/incGenFromIpDesc.html" at <readFile $path>: error calling readFile: stat /home/vsts/work/1/s/build/docs-generated/hw/ip/sysrst_ctrl/data/sysrst_ctrl_testplan.hjson.testplan: no such file or directory
Error building site
```

The reason is this comment on line 122 needs to be satisfied. Please update `util/build_docs.py` and add entry in there so that it can find the sysrst CTRL testplan correctly.